libs/features/components/cards/src/lib/cards/elewa-vertical-icon-and-text/elewa-vertical-icon-and-text.component.html

```diff
@@ -1,5 +1,11 @@
-<div class="about-card">
-  <i class="{{ icon }} about-icon"></i>
-  <h3 class="about-title">{{ title }}</h3>
-  <p class="about-description">{{ description }}</p>
+<div class="cards-main">
+  <div class="card" *ngFor="let property of properties" let i = index>
+    <div class="about-icon">
```

Comment:
You've edited a different issue!!
This is a reusable component
The loop should happen on your component, not here.